TarponSimulator2017/Core/Physics.cs

```diff
@@ -59,20 +59,16 @@ public Physics ()
 			ApplicationPoint = new Vector2 (AbscisseDepart, OrdonneeDepart);
 
 			Orientation = new Vector2 (0, -1);
-
-
-
 		}
 
 		public float OrientiationFloat ()
 		{
-			return (float)Math.Atan2 (Orientation.X, -Orientation.Y);
+			return (float) (Math.Atan2 (Orientation.X, -Orientation.Y) + Math.PI);
```

Comment:
Why add Math.PI ?

Reply:
Nice shot!
The explanation is a bit long but interesting for furture development.
I use the output of `OrientiationFloat` to compute the rotation of the sprite (the png representing the boat).
So when `OrientiationFloat` output a value of `0f`, the sprite is not rotated.
But `Math.Atan2 (Orientation.X, -Orientation.Y)` equals 0 when `x = 0, y = -1`.
So, our boat will move to the top when we will accelerate with a rotation of 0.
The previous sprite was a boat pointing to the top, so no problem here.
But the [new spritesheet](https://github.com/Hamilcare/TarponSimulator2017/blob/cbefbac4b4f71b953b5e85c6c9f696b2fb0de9e8/TarponSimulator2017/Content/img/spritesheet.png) has ships stored pointing to the bottom of the screen.
So, to draw a ship that points to the top, I must rotate it by `90 deg` or `Math.PI rad`.

After that small explanation, I am realizing that this shift should be put in the Draw part, as it is sprite dependant. I will fix that as soon as possible.

Reply:
Thanks for the explanation, that makes sense :)